src/commons/api/history.rs

```diff
@@ -174,24 +176,24 @@ impl CommandHistoryRecord {
 //------------ StoredEffect --------------------------------------------------
 
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
-#[serde(rename_all = "snake_case")]
+#[serde(rename_all = "snake_case", tag = "result")]
```

Comment:
I haven't checked all of the OpenAPI spec definitions that might be affected by the introduction of serde "internal tagging", but as you are intend to remove "embedded" which will also impact the OpenAPI spec definitions I suggest we review it fully then. @timbru: Do we need a specific GH issue to remind us of this specific aspect to check in the OpenAPI spec definitions or do you have that covered already?